en/peripherals/dshot.md

```diff
@@ -0,0 +1,79 @@
+# DShot ESCs
+
+DShot is an alternative ESC protocol that has several advantages over PWM or OneShot:
+- Reduced latency
+- Checksum (increased robustness)
+- Digital encoding (no need for ESC calibrations)
+- Possibility to get feedback (telemetry)
+- Sending commands (such as reversing the motor spin direction)
+
+DShot ESCs are connected and wired the same way as [PWM ESCs](pwm_escs_and_servo.md), and you can switch between these protocols just by changing software parameters (ESCs automatically detect the selected protocol on startup).
+
+## Configuration
+
+> **Warning** Remove propellers before changing ESC configuration parameters!
+
+Enable DShot with the [DSHOT_CONFIG](../advanced_config/parameter_reference.md#DSHOT_CONFIG) parameter (if the parameter does not exist, the board does not support DShot).
+
+DShot comes with different speed options: *DShot150*, *DShot300*, *DShot600* and *DShot1200*, where the number indicates the speed in bits per second.
```

Comment:
Is it bits per second? You other docs indicate kb/s

Reply:
@bkueng ^^^

Reply:
Hmm I thought I replied here as well.
Anyway, the speed would be `150'000' b/s, which is shortened in the name as `DShot150`. So kb/s would be more appropriate here indeed.

Reply:
Thanks @bkueng . Who knows - sometimes github does odd shit :-)

I changed this to kilobits/second. I know that contracts to Kb, but for some reason I always look at that and think - typo, they mean KB (kilobytes).